app/Options.hs

```diff
@@ -16,11 +22,12 @@ data Options
       }
 
 data Backend
-  = Unit
-  | Naturals
-  | Michelson
+  = Michelson BMichelson
+  -- | Plonk (BPlonk Fr)
```

Comment:
Probably for this branch, best to just remove it, then get it back in your plonk branch